torchaudio/sox_effects/sox_effects.py

```diff
@@ -14,6 +14,7 @@ def init_sox_effects():
 
     Note:
         You do not need to call this function manually. It is called automatically.
+        This function only works on CPU Tensors.
```

Comment:
This is not the right function. The note has to be added to `apply_effects_tensor` function.

Reply:
Ah got a bit confused. Should have read your comments carefully.